### ISSUE TITLE
show information on the image

### ISSUE THREAD

**Issue by louzhu233** [AlexeyAB/darknet#683]
@AlexeyAB 
  Hi!AlexeyAB ,I am going to show some information on the detected image for the given label，how should I do? Thanks

**Comment by AlexeyAB** [comment]
Hi,

To draw something during **detection on video**, you can add here your code using common OpenCV functions: https://github.com/AlexeyAB/darknet/blob/a2f3644e87c11c222f86aa2aeb0e31d858ebb024/src/image.c#L350

For example:
```cpp
int width = 3;
cvRectangle(show_img, cvPoint(0, 0), cvPoint(20,20), CV_RGB(10, 100, 200), width, 8, 0);
CvScalar black_color;
black_color.val[0] = 0;
float const font_size = 1;
CvFont font;
cvInitFont(&font, CV_FONT_HERSHEY_SIMPLEX, font_size, font_size, 0, font_size * 3, 8);
cvPutText(show_img, "my text", cvPoint(50,50), &font, black_color);
```

**Comment by louzhu233** [comment]
@AlexeyAB 
  Thanks for your reply.I want to draw the information for a specific label.Should I use the code
if (label==**) or if (class_id==**) or something else.Thanks

**Comment by AlexeyAB** [comment]
Yes, use `if(strcmp(labelstr, "somelabel")==0) {  }` or `if(class_id == 11) {  }`
And add your code here: https://github.com/AlexeyAB/darknet/blob/a2f3644e87c11c222f86aa2aeb0e31d858ebb024/src/image.c#L369

**Comment by louzhu233** [comment]
@AlexeyAB 
  I add the codes after the code  if (class_id >= 0) { ...  as follows:
 	
> 	if (class_id >= 0) {
> 			int width = show_img->height * .006;
>                         if (class_id == 6) {
>                                 int width = 3;
>                                 cvRectangle(show_img, cvPoint(0, 0), cvPoint(20,20), CV_RGB(10, 100, 200), width, 8, 0);
>                                 CvScalar black_color;
>                                 black_color.val[0] = 0;
>                                 float const font_size = 1;
>                         CvFont font;
>                          
>                         cvInitFont(&font, CV_FONT_HERSHEY_SIMPLEX, font_size, font_size, 0, font_size * 3, 8);
>                         
>                                 cvPutText(show_img, "attention", cvPoint(50,50), &font, black_color);}
  but when I test again,the information "attention" doesn't appear on the image.Is there something wrong?Thanks very much!

**Comment by AlexeyAB** [comment]
It works only for video. If you test using image - it willn't appear.

**Comment by louzhu233** [comment]
@AlexeyAB 
 So nice of you!Now I had tested it for video.But there is another question.The box appears in the upper left corner of the video but the information doesn't appear.What's wrong?And how to change the code if I want the information appear in the lower left corner of the video? Thanks

**Comment by louzhu233** [comment]
@AlexeyAB 
Thank you so much!I had solved it
